portals/new_admin/portals/client/dist/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>React Webpack App</title>
+</head>
+
+<body>
+    <div id="root"></div>
+    <script src="bundle.js"></script>
+</body>
+
+</html>
```

Comment:
Shall we add an EOF new line here as well?

Reply:
Currently, there is no such file.